.travis.yml

```diff
@@ -39,6 +39,16 @@ script:
       mvn clean install -B -U
     fi
 
+after_success:
+  # hide sensitive data from codecov-io
+  - rm -f config/travis-app-maven-plugin-e1bb638f290e.json
+  - export SONATYPE_USERNAME=[secure]
```

Comment:
why '[secure]'? I know that's the string that travis prints out when it assigns a secret value, but in our case we want to wipe it out.